### ISSUE TITLE
Accessibility : Videos:Accessibility : Videos: Current/total time of video announced without context and with a button role 

### ISSUE THREAD

**Issue by himanshikhattar** [eBay/ebayui-core#1578]
# Bug Report

1. **Using voice over**, view video
2. Open the video and pause. The focus is on the play button. Use ctrl+option+right arrow key to navigate to the next item. The focus is on the current/total time and note the announcement.

Expected Result
The current/total time should be announced in context, e.g. as 'time played of time total, <value_played / <value_total>'.

Actual Result
The current/total time is announced without any context as '0:01 / 0:15'.
Also, it is announced as 'button, dimmed'.

## Screenshots
<img width="1792" alt="Screen Shot 2021-12-06 at 2 44 26 PM" src="https://user-images.githubusercontent.com/95649085/144937865-9352e276-6859-40fd-a8cc-a9d33d3775ae.png">

**Comment by ianmcburnie** [comment]
@himanshikhattar Thank you for considering accessibility :-)

Generally speaking we try and use the same text for all users wherever possible, unless we feel it could be very misleading, confusing and/or lead to an unintended action (a great example is prefixing "Strikethrough price" before a sale price due to screen readers not conveying the visual strikethrough).

I think this UI is probably fine as is, given the context of a video player, the proximity to other video controls, and the universally recognized time format that is being used. I just checked YouTube for a quick comparison and they also omit any extra text (that's not to say other sites are always right, mind you). I'll check in with the rest of the a11y team and see what they think.

The fact that this has been marked up as a button is odd. However, this is 3rd party code so the fix to change it from button to static text would need to be made on their end. We can open a ticket.

**Comment by ianmcburnie** [comment]
Closing this based on reasoning above and further discussion with team. Also a reminder that because this is presented as a recognizable time format, some screen readers will attempt to read parse it and read it in a more friendly way depending on users punctuation/verbosity settings.
